src/rad/mainframe.cpp

```diff
@@ -968,6 +968,10 @@ void MainFrame::UpdateFrame()
 	wxMenu* menuEdit = GetMenuBar()->GetMenu( GetMenuBar()->FindMenu( wxT( "Edit" ) ) );
 	wxToolBar* toolbar = GetToolBar();
 
+	toolbar->EnableTool(ID_SAVE_PRJ, AppData()->IsModified());
+	wxMenu* menuFile = GetMenuBar()->GetMenu(GetMenuBar()->FindMenu(_("File")));
+	menuFile->Enable(ID_SAVE_PRJ, AppData()->IsModified());
```

Comment:
I don't want to nitpick too much but looking at the whole section of that file i can see that there is a certain order of statements and this order is violated here.

It would look more uniform if the retrieval of the file menu would be right above the retrieval of the edit menu and if you first enable the menu entry and then the toolbar entry. Can you reorder the statements please?

Reply:
Since the rest of that code block is using **toolbar**, I moved that just under the comment that starts the section, and then moved the File enabling code above the edit menu section so that the code flows in a more expected way.

Reply:
BTW, I just want to mention that I appreciate the feedback you are giving on these PRs. I'm new to the project, and your comments will help a lot in my getting a better understanding of what you're looking for in coding changes. I'd rather get a thorough handle on the preferred coding styles and practices on these minor PRs before I dive into anything more complex.

Reply:
These changes work for me. Now there is just one final step left i have to ask you for. All these fixup commits just clutter up the history, can you squash all these commits into one and force-push them? I could squash them on merge but i'd like to keep the branch visible in the history.

You are doing quite a lot of commits so can i ask you to take a look at https://chris.beams.io/posts/git-commit/ and take it as a guidance for your commit messages? Especially use the imperative form for the things you changed but not for the explanations why (see your other outstanding PR). As an example, in this PR the message would turn from `Enables/Disables Save button based on modified state of project file.` into `Enable/Disable save action based on modified state of project file`